biocrnpyler/dna_part.py

```diff
@@ -0,0 +1,70 @@
+################################################################
```

Comment:
There's a standard header on the top of the new files with licence info, please add it.

Reply:
where can i find this header?

Reply:
Here you go:
# Copyright (c) 2020, Build-A-Cell. All rights reserved.
# See LICENSE file in the project root directory for details.